tests/ui/pages/desktop/base.py

```diff
@@ -26,6 +26,15 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    def wait_for_page_to_load(self):
+        body = self.find_element(By.TAG_NAME, 'body')
+        self.wait.until(lambda s: 'firefox' in body.get_attribute('class'))
```

Comment:
My point is that if a page object extends `Base` (which they all should) then it will inherit this wait, which is not suitable for Thunderbird pages for example. This might be fine in `Home` where the default product is Firefox. GeckoDriver will eventually wait after a click, once it's spec compliant, but that doesn't mean we'll be able to remove waiting for page loads because it won't know what to wait for.. it'll just wait for general page ready state, and more specific waits for individual page objects will still be a good practice.

Perhaps ask the dev team if there's something we can wait for to indicate the page load is complete. For example, bedrock wait for the current URL to match the see URL, and then for the `html` element to have a `loaded` class: https://github.com/mozilla/bedrock/blob/master/tests/pages/base.py#L32-L34